### PR TITLE
read "/proc/meminfo" to get memory status

### PR DESCRIPTION
**Pull request by cwl233**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug

**What this PR does / why we need it**:
If you run edgecore on ubuntu whose system language is not English, you will not get the edge memory correctly, like this:
Capacity:
 cpu:     80
 memory:  0
 pods:    110
Allocatable:
 cpu:     80
 memory:  0
 pods:    110
Because edged use command "free -m" to get the capacity of memory.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by kadisi** [comment]
that  looks good by read `/proc/meminfo`.

File: edge/pkg/edged/edged_status.go [hunk 1]
```diff
@@ -24,10 +24,13 @@ package edged
 
 import (
 	"fmt"
+	"io/ioutil"
 	"net"
 	"os"
 	"regexp"
 	"runtime"
+	"strconv"
+
```
Comment (fisherxu) [inline edge/pkg/edged/edged_status.go]: delete the blank line here

**Comment by fisherxu** [comment]
There's some vendor check err, can run 'hack/update-vendor.sh' to update the vendor :-). @cwl233

**Comment by cwl233** [comment]
> There's some vendor check err, can run 'hack/update-vendor.sh' to update the vendor :-). @cwl233

i deleted it

**Comment by kadisi** [comment]
@cwl223 please rebase your commit

**Pull request merged**